Source/Core/Core/GeckoCodeConfig.cpp

```diff
@@ -2,17 +2,173 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#include "Core/GeckoCodeConfig.h"
+
 #include <algorithm>
 #include <sstream>
```

Comment:
`<cstddef>` needs to be included (for `size_t`), and `<memory>` needs to be included (for `unique_ptr`)